modules/ref_fb_module/src/renderer_fb_impl.cpp

```diff
@@ -727,6 +730,8 @@ void RendererFbImpl::renderLoop()
     auto waitTime = defaultWaitTime;
     while (!stopRender && window.isOpen())
     {
+        setComponentStatus(ComponentStatus::Ok);
```

Comment:
Please pay attention to this method call during the review, as it may be problematic. The reason being that we do this inside a loop (the `renderLoop`), which might result in Component status being changed many times, if something is wrong. E.g. this will reset it to `Ok` every time, then inside the loop the status might change again during execution (again multiple times for each loop iteration). Which in turn might cause the status to "blink" in the user interface if implemented. One solution is to put it outside of the loop, but then it won't reset once set. Also whatever Warning/Error might happen inside the loop will repeat multiple times regardless.

Reply:
Maybe you can introduce internal member that remembers the last state and only change the state when current state is different than previous.